spec/initializers/yaml_autoloader_spec.rb

```diff
@@ -6,6 +6,7 @@
   let(:missing_model)   { model_directory.join("zzz_model.rb") }
 
   before do
+    skip "This is currently not testable with zeitwerk!" if Rails.application.config.autoloader == :zeitwerk
```

Comment:
I tried many things to test this with zeitwerk but either can't modify the autoload paths OR, it's too late and the changes to these paths aren't used.  Basically, we can't do proper setup/teardown to test this with zeitwerk.  Cross repo fails in `manageiq-providers-vmware` if you remove the yaml autoloader, see above.